hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteOperatorCoordinator.java

```diff
@@ -401,4 +422,26 @@ public OperatorCoordinator create(Context context) {
       return new StreamWriteOperatorCoordinator(this.conf, context);
     }
   }
+
+  /**
+   * Remember some table state variables.
+   */
+  private static class TableState implements Serializable {
```

Comment:
Shall we make this class follow the POJO specific?

Reply:
No need to do that because this is a inner class.

Reply:
OK, I originally deleted this comment, but it was unsuccessful due to network problems.